br/pkg/lightning/config/config.go

```diff
@@ -937,7 +937,7 @@ func NewConfig() *Config {
 		Cron: Cron{
 			SwitchMode:     Duration{Duration: DefaultSwitchTiKVModeInterval},
 			LogProgress:    Duration{Duration: 5 * time.Minute},
-			CheckDiskQuota: Duration{Duration: 1 * time.Minute},
+			CheckDiskQuota: Duration{Duration: 10 * time.Second},
```

Comment:
we don't need to change this, user can config on their need, we have doc about it

https://docs.pingcap.com/tidb/dev/tidb-lightning-physical-import-mode-usage